src/wallet/rpcwallet.cpp

```diff
@@ -3663,6 +3663,7 @@ UniValue getaddressinfo(const JSONRPCRequest& request)
                         {RPCResult::Type::BOOL, "iswatchonly", "If the address is watchonly."},
                         {RPCResult::Type::BOOL, "solvable", "If we know how to spend coins sent to this address, ignoring the possible lack of private keys."},
                         {RPCResult::Type::STR, "desc", /* optional */ true, "A descriptor for spending coins sent to this address (only when solvable)."},
+                        {RPCResult::Type::STR, "parent_desc", /* optional */ true, "The descriptor used to derive this address if this is a descriptor wallet"},
```

Comment:
e4ac869

- missing period like the neighboring sentences and maybe add a comma
```suggestion
                        {RPCResult::Type::STR, "parent_desc", /* optional */ true, "The descriptor used to derive this address, if this is a descriptor wallet."},
```

- not sure, but maybe change from:

  "The descriptor used to derive this address, if this is a descriptor wallet."

  to

  "The descriptor used to derive this address, if generated by a descriptor wallet."